mapcompose/src/main/java/ovh/plrapps/mapcompose/api/PathApi.kt

```diff
@@ -18,6 +18,7 @@ import ovh.plrapps.mapcompose.ui.state.MapState
  * @param pathData Obtained from [PathDataBuilder.build]
  * @param width The width of the path, in [Dp]. Defaults to 4.dp
  * @param color The color of the path. Defaults to Color(0xFF448AFF)
+ * @param fillColor If set - the path will be closed and the area drawn in this color.
```

Comment:
Small change:
`@param fillColor If set - the path will be filled and the area drawn in this color.`